### PR TITLE
Hello Microverse

### PR DESCRIPTION
**Pull request by susangoga**
# Hi good reviewer!
## These are the changes proposed for the project:
- Document work in a professional way.
- Create a new GitHub repo.
- Create a new branch according to the GitHub flow rules.
- Add a .gitignore file.
- Set up a linter for HTML and CSS.
- Add descriptive README file.
- Add 3 files:

### CODE REVIEW

File: .github/workflows/linters.yml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+      - name: Stylelint Report
+        run: npx stylelint "**/*.{css,scss}"
+  nodechecker:
+    name: node_modules checker
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@v2
+      - name: Check node_modules existence
+        run: |
+          if [ -d "node_modules/" ]; then echo -e "\e[1;31mThe node_modules/ folder was pushed to the repo. Please remove it from the GitHub repository and try again."; echo -e "\e[1;32mYou can set up a .gitignore file with this folder included on it to prevent this from happening in the future." && exit 1; fi
```
Comment (Reem-lab) [inline .github/workflows/linters.yml:56]: - [ ] Great job on adding the linters file but you need also to add the files for `hintrc` to check for HTML linters and `stylelint` to check for CSS linters so kindly add them to your project 

image

- [ ] I think you didn't install the packages for hintrc and stylelint so kindly if you scroll down in this [Linters for HTML and CSS](https://github.com/microverseinc/linters-config/tree/master/html-css)you will see some steps on how to install them then commands to check if there's any linters error 🔥🚀💯

image


## IF you scroll dow you will see the commands so you can check if there's any linters error need to fix 💯☺💪🏻
- This Command to check for HTML linters errors
image

- This Command to check for CSS linters errors 💯
image

**Review by Reem-lab — CHANGES_REQUESTED** [review]
# Hi @Zuz4n, 👋🏻👋🏻 

Good job so far! 💯🚀💪🏻🔥🔥
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Great working on the set up the project ✅
📌 Good commit messages ✅


## Required Changes ♻️
_Check the comments under the review._
- [ ] Great job but I think you forget to customzide Microverse README [template](https://github.com/microverseinc/readme-template) and remembre it is very important because it showcase your project and what is about? and some steps about how to clone the project if someone want to clone it and it shows who contributed with this project? so that's very important in the future when searching for a job so kindly add it as required 💪🏻☺💯
image

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: .github/workflows/linters.yml (hunk 1 above)
Comment (susangoga) [reply .github/workflows/linters.yml:56]: Hello Rem-Lab

I resolved the required changes, my question is: Do I need to ask for a new review?, Sorry, I'm still learning...

File: README.md [hunk 2]
```diff
@@ -1 +1,20 @@
...
+
+🤝 Contributing
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the issues page.
+
+*_Show your support_*
+Give a ⭐️ if you like this project!
+
+📝 License: MIT licensed.
```
Comment (fherrerao) [inline README.md]: - [ ] Good job adding information to your `Readme` file as requested by the previous reviewer 😺, but it should look professional. Kindly, use this [template](https://github.com/microverseinc/readme-template/blob/master/README.md) and adapt it with information related to your project. 🦖

**Review by fherrerao — CHANGES_REQUESTED** [review]
Hi @Zuz4n, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

So Close Snl GIF by Saturday Night Live
 ## Highlights
✅ No linter errors.
✅ Great Gitub flow.
✅ Excellent commit messages.

 ## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions

- [ ] [OPTIONAL] The branch name is important because it gives an idea to know what are you doing in the branch 💯 . The branch should have a name of a feature or action (e.g. 'created_sidebar' or 'added_comments_section'). Please rename it 🥇 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request closed without merging**